lib/trailblazer/operation.rb

```diff
@@ -45,6 +45,18 @@ def present(*params)
       def contract(&block)
         contract_class.class_eval(&block)
       end
+      
+      def collection(*params, &block)
```

Comment:
Benefits of this API:

1. We can handle in different ways a single model and collection (duh!), so we don't need to handle using `ifs` each case
2. In collections we can set [scope](https://github.com/apotonick/trailblazer/pull/40/files#diff-c518d16668a2dc2a2951d3206f055936) and [pagination](https://github.com/apotonick/trailblazer/pull/40/files#diff-9355b6bc2d6c54ad4882dcf23cbff87a) out-of-the-box, then we just need to include [Scope](https://github.com/apotonick/trailblazer/pull/40/files#diff-6e0b5532e3691435735d5f7ebe67843eR7) and [Pagination](https://github.com/apotonick/trailblazer/pull/40/files#diff-0cfc7048e88372dae0e3076e477ef5e0R7) (links are for tests) on Collection operations
3. in crud.rb we can set [collection variables](https://github.com/apotonick/trailblazer/pull/36/files#diff-df1b16d18715ddbab4a106cee89a9b4cR100)